ush/generate_FV3LAM_wflow.sh

```diff
@@ -458,6 +458,10 @@ settings="\
   'ensctrl_stmp': ${ENSCTRL_STMP}
   'use_rrfse_ens': ${USE_RRFSE_ENS}
 #
+# smoke and dust related parameters.
+#
+  'do_smoke_dust': ${DO_SMOKE_DUST}
+#
```

Comment:
It looks like "do_smoke_dust" is not in the FV3LAM_wflow.xml file and hence at this time we don't need Line 461-464 in ush/generate_FV3LAM_wflow.sh.